forge-gui/res/cardsfolder/l/loose_in_the_park.txt

```diff
@@ -0,0 +1,11 @@
+Name:Loose in the Park
+ManaCost:1 G
+Types:Enchantment Aura
+K:Enchant land
+A:SP$ Attach | Cost$ 1 G | ValidTgts$ Land | AILogic$ Pump
```

Comment:
You can remove Cost param from the main SP$ line if it is the same as ManaCost line, as is the case here